content/topics/linux/intro-to-linux/_index.md

```diff
@@ -2,6 +2,6 @@
 title: Introduction to Linux
 ---
 
-Linux is an operating system (much like windows is an operating system). Linux started of as one nerd's hobby but quickly grew in fame. Linux is free, it's source code is freely available, and it is created, developed and used by a large community
+* Linux is an operating system (much like windows is an operating system). Linux started of as one nerd's hobby but quickly grew in fame. Linux is free, it's source code is freely available, and it is created, developed and used by a large community
 
-https://www.youtube.com/watch?v=PTaL1s3YJPY
+- https://drive.google.com/file/d/1kXD9aTjsc1zVFY_HetFwDm9J2XMdmr1E/view?usp=sharing
```

Comment:
instead of just putting in a url can you do something like this: https://master.d3hv5c7o5j7vym.amplifyapp.com/workshops/how-to-be-a-professional/

How to get the iframe:
- open the presentation in google slides
- click file > publish to the web
- click "embed"